### PR TITLE
added meme package 

### PR DESCRIPTION
**Pull request by JusticeForMikeBrown**
```
user@node103[~/repos/spack]$ which meme
/pbtech_mounts/homes027/dod2014/repos/spack/opt/spack/linux-rhel6-x86_64/gcc-4.4.6/meme-4.11.4-n3oeo7iu7w3hfzatl2jqdiqhssfkbnqy/bin/meme

user@node103[~/repos/spack]$ meme -version
4.11.4

```

http://meme-suite.org/doc/install.html?man_type=web#quick

### CODE REVIEW

File: var/spack/repos/builtin/packages/meme/package.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    """The MEME Suite allows the biologist to discover novel motifs in
+    collections of unaligned nucleotide or protein sequences, and to perform a
+    wide variety of other motif-based analyses."""
+
+    homepage = "http://meme-suite.org"
+    url      = "http://meme-suite.org/meme-software/4.11.4/meme_4.11.4.tar.gz"
+
+    version('4.11.4', '371f513f82fa0888205748e333003897')
+
+    depends_on('zlib', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/meme/package.py]: zlib isn't usually a build/run dependency. Did you have problems building meme when it was the default dependency type?

Comment (JusticeForMikeBrown) [inline var/spack/repos/builtin/packages/meme/package.py]: I didn't try.   It lists zlib as a dependency so I added it here.

Comment (adamjstewart) [inline var/spack/repos/builtin/packages/meme/package.py]: Oh I agree that it's a dependency, but I think the dependency type is wrong. If you don't know what a dependency type is, you should read the Packaging Guide.

Comment (JusticeForMikeBrown) [inline var/spack/repos/builtin/packages/meme/package.py]: Read as suggested.

It's only a link dependency:

```
user@node103[~/repos/spack]$ ldd -r $(which meme)                                                                                                                                                                                          
        linux-vdso.so.1 =>  (0x00007fff961ff000)
        libxslt.so.1 => /usr/lib64/libxslt.so.1 (0x0000003dd0200000)
        libexslt.so.0 => /usr/lib64/libexslt.so.0 (0x0000003dd1a00000)
        libxml2.so.2 => /usr/lib64/libxml2.so.2 (0x0000003dc3400000)
        libz.so.1 => /pbtech_mounts/homes027/dod2014/repos/spack/opt/spack/linux-rhel6-x86_64/gcc-4.4.6/zlib-1.2.11-erzei6xutxne2tjssbmelgutsnhz6x6s/lib/libz.so.1 (0x00007f0187c9a000)
        libm.so.6 => /lib64/libm.so.6 (0x0000003db9c00000)
        libc.so.6 => /lib64/libc.so.6 (0x0000003db9400000)
        libgcrypt.so.11 => /lib64/libgcrypt.so.11 (0x0000003dcbe00000)
        libdl.so.2 => /lib64/libdl.so.2 (0x0000003db9000000)
        libgpg-error.so.0 => /lib64/libgpg-error.so.0 (0x0000003dcd600000)
        /lib64/ld-linux-x86-64.so.2 (0x0000003db8c00000)
```

Comment (adamjstewart) [inline var/spack/repos/builtin/packages/meme/package.py]: Oh, it looks like you're missing several other dependencies as well. Try adding dependencies on libxml2 and libxslt. Possibly openssl? I forget where libgcrypt comes from.

Comment (JusticeForMikeBrown) [inline var/spack/repos/builtin/packages/meme/package.py]: Thanks!!!!

```
user@node103[~/repos/spack]$ rpm -qf /lib64/libgcrypt.so.11
libgcrypt-1.4.5-9.el6_2.2.x86_64

```

Comment (JusticeForMikeBrown) [inline var/spack/repos/builtin/packages/meme/package.py]: user@node103[~/repos/spack]$ ldd -r $(which meme)   
        linux-vdso.so.1 =>  (0x00007fff777ff000)
        libxslt.so.1 => /pbtech_mounts/homes027/user/repos/spack/opt/spack/linux-rhel6-x86_64/gcc-4.4.6/libxslt-1.1.29-44723n7a6zoh3hw3h2epwsvwetnaxkpt/lib/libxslt.so.1 (0x00007fdd19c29000)
        libexslt.so.0 => /pbtech_mounts/homes027/user/repos/spack/opt/spack/linux-rhel6-x86_64/gcc-4.4.6/libxslt-1.1.29-44723n7a6zoh3hw3h2epwsvwetnaxkpt/lib/libexslt.so.0 (0x00007fdd19a14000)
        libxml2.so.2 => /pbtech_mounts/homes027/user/repos/spack/opt/spack/linux-rhel6-x86_64/gcc-4.4.6/libxml2-2.9.4-szkxoqturs2fzrvsrzz4iolbnl33qjdf/lib/libxml2.so.2 (0x00007fdd196b7000)
        libz.so.1 => /pbtech_mounts/homes027/user/repos/spack/opt/spack/linux-rhel6-x86_64/gcc-4.4.6/zlib-1.2.11-erzei6xutxne2tjssbmelgutsnhz6x6s/lib/libz.so.1 (0x00007fdd19499000)
        libm.so.6 => /lib64/libm.so.6 (0x0000003db9c00000)
        libc.so.6 => /lib64/libc.so.6 (0x0000003db9400000)
        liblzma.so.5 => /pbtech_mounts/homes027/user/repos/spack/opt/spack/linux-rhel6-x86_64/gcc-4.4.6/xz-5.2.3-7ah2pytuetgk63x5scnbdoma63lvijvz/lib/liblzma.so.5 (0x00007fdd19248000)
        libdl.so.2 => /lib64/libdl.so.2 (0x0000003db9000000)
        librt.so.1 => /lib64/librt.so.1 (0x0000003dba400000)
        libpthread.so.0 => /lib64/libpthread.so.0 (0x0000003db9800000)
        libgcrypt.so.20 => /pbtech_mounts/homes027/user/repos/spack/opt/spack/linux-rhel6-x86_64/gcc-4.4.6/libgcrypt-1.6.2-icbvp2oy6amw4epfjbm442rwyejjodcw/lib/libgcrypt.so.20 (0x00007fdd18f81000)
        libgpg-error.so.0 => /pbtech_mounts/homes027/user/repos/spack/opt/spack/linux-rhel6-x86_64/gcc-4.4.6/libgpg-error-1.21-jzkx2yr3ctlxks525xo6yo2xap6ievnj/lib/libgpg-error.so.0 (0x00007fdd18d6f000)
        /lib64/ld-linux-x86-64.so.2 (0x0000003db8c00000)

**Comment by JusticeForMikeBrown** [comment]
@adamjstewart 

Thanks again! 

I built new version of R (3.4.0) so will submit another PR once this has been closed.

Best
Doug

**Comment by adamjstewart** [comment]
@JusticeForMikeBrown For your next PR, try creating a separate branch for any changes to R so you don't have a PR with 17 commits like this one.

**Pull request merged**